Apromore-Extras/APMLogModule/src/test/java/org/apromore/apmlog/customcalendartests/CusCalArcDurFIlterTest.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * #%L
+ * This file is part of "Apromore Enterprise Edition".
+ * %%
+ * Copyright (C) 2019 - 2021 Apromore Pty Ltd. All Rights Reserved.
+ * %%
+ * NOTICE:  All information contained herein is, and remains the
+ * property of Apromore Pty Ltd and its suppliers, if any.
+ * The intellectual and technical concepts contained herein are
+ * proprietary to Apromore Pty Ltd and its suppliers and may
+ * be covered by U.S. and Foreign Patents, patents in process,
+ * and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this
+ * material is strictly forbidden unless prior written permission
+ * is obtained from Apromore Pty Ltd.
+ * #L%
+ */
+package org.apromore.apmlog.customcalendartests;
+
+import org.apromore.apmlog.APMLog;
+import org.apromore.apmlog.filter.APMLogFilter;
+import org.apromore.apmlog.filter.rules.LogFilterRule;
+import org.apromore.apmlog.filter.rules.LogFilterRuleImpl;
+import org.apromore.apmlog.filter.rules.RuleValue;
+import org.apromore.apmlog.filter.types.Choice;
+import org.apromore.apmlog.filter.types.FilterType;
+import org.apromore.apmlog.filter.types.Inclusion;
+import org.apromore.apmlog.filter.types.OperationType;
+import org.apromore.apmlog.filter.types.Section;
+import org.apromore.apmlog.stats.CustomTriple;
+import org.apromore.apmlog.util.Util;
+import org.apromore.calendar.builder.CalendarModelBuilder;
+import org.apromore.calendar.model.CalendarModel;
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class CusCalArcDurFIlterTest {
```

Comment:
Surely this should be `CusCalArcDurFilterTest'?

Reply:
Now that I look at it more closely (since SonarCloud pointed it out) this probably should either be named something just *Utility.java rather than *Test.java (since the latter implies it's a test suite) or simply be a private method of APMLogUnitTest if it's unlikely to be reused.